cita-chain/core/src/snapshot/service.rs

```diff
@@ -42,10 +41,7 @@ use libchain::chain::{get_chain, get_chain_body_height, Chain};
 use filters::PollManager;
 
 /// Number of blocks in an ethash snapshot.
-// make dependent on difficulty incrment divisor?
-//const SNAPSHOT_BLOCKS: u64 = 5000;
 /// Maximum number of blocks allowed in an ethash snapshot.
-//const MAX_SNAPSHOT_BLOCKS: u64 = 30000;
 
```

Comment:
```
/// Number of blocks in an ethash snapshot.
/// Maximum number of blocks allowed in an ethash snapshot.
```
These two lines have extra usage?

Reply:
NO

Reply:
if no extra consideration, may be we can del it by the way.

Reply:
done